lib/src/modules/common/file_upload/file_uploader.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:fa_flutter_core/fa_flutter_core.dart';
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:open_file/src/platform/open_file.dart';
+
+class FileService {
+  FileService._();
+
+  /// Return the Clicked Image
+  static Future<PlatformFile?> pickImage(
+      ImageSource source, StateSetter setState) async {
+    final pickedFile = await ImagePicker().pickImage(source: source);
+
+    if (pickedFile != null) {
+      return PlatformFile(
+        name: "uploaded_image",
+        size: 5,
+        path: pickedFile.path,
+      );
+    }
+    return null;
+  }
+
+  /// Return the Picked File from the System
+  static Future<PlatformFile?> pickDocument(StateSetter setState) async {
+    FilePickerResult? result = await FilePicker.platform.pickFiles(
+      type: FileType.custom,
+      allowedExtensions: ['pdf', 'jpg', 'jpeg', 'png', 'gif'],
```

Comment:
can we have list of allowedExtensions as nullable optional variable in the parameter, because in case if we require extra functionality we will pass it into the function.

also mention the allowed extensions in the Doc Comments